request.c

```diff
@@ -441,7 +460,7 @@ void getTopClients(char *client_message, int *sock)
 			}
 		}
 
-		if(clients[j].count > 0)
+		if(includezeroclients || clients[j].count > 0)
```

Comment:
Return this client if either

1. `withzero` option is set, and/or
2. the client made at least one query within the most recent 24 hours